woostarter/patterns/about-2-columns-with-heading.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Title: About 2 columns with heading
+ * Slug: woostarter/about-2-columns-with-heading
+ * Categories: text, about, media
+ * Keywords: about, text
+ * Description: A section with two columns of text, heading and an image.
+ *
```

Comment:
I've noticed that a bunch of the patterns have this empty line in the meta section. It doesn't really make a difference, other than the consistency of the way the patterns are built. Happy to go with whatever you prefer but I think it'd be nice if all patterns had consistent a meta section - probably something we can handle in a follow-up closer to launch.